core/scripts/keystone/src/testdata/SampleConfig.json

```diff
@@ -5,6 +5,7 @@
     "MaxReportLengthBytes": 1000000,
     "MaxRequestBatchSize": 1000,
```

Comment:
Should be MaxBatchSize. Can you do a search across the repo for other possible missed occurrences (in Keystone)?